### PR TITLE
docs: add v9 migration

### LINKED ISSUE #2216: Docs: add migration guide for version 9 release
Documents

- [ ] CLI Update instructions
- [ ] New Features/Breaking Changes
- [ ] Provided Migrations
- [ ] Latest dependencies

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #2216 

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/migration/v9.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Version 9 has the minimum version requirements:
+
+- Angular version 9.x
+- Angular CLI version 9.x
+- TypeScript version 3.7.x
+- RxJS version 6.5.x
+
+<div class="alert is-helpful">
+
+NgRx version 9 can be used with earlier versions of Angular. TK
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v9.md]: This should be removed. Even though this is true, we don't want to officially support doing this.

File: projects/ngrx.io/content/guide/migration/v9.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+The event on the payload of the dispatched actions will only contain the router event id and the url, instead of the Angular RouterEvent.
+
+<div class="alert is-helpful">
+
+A migration is provided and adds the `DefaultRouterStateSerializer` if a serializer isn't set.
+
+</div>
+
+### @ngrx/schematics
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v9.md]: We need to add a section for `minimal` with ng-add for `@ngrx/store` also.

```suggestion
### @ngrx/effects
```

File: projects/ngrx.io/content/guide/migration/v9.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
+# V9 Update Guide
+
+## Angular CLI update
+
+NgRx supports using the Angular CLI `ng update` command to update your dependencies. Migration schematics are run to make the upgrade smoother. These schematics will fix breaking changes.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v9.md]:
```suggestion
NgRx supports using the Angular CLI `ng update` command to update your dependencies. Migration schematics are run to make the upgrade smoother. These schematics will fix some of the breaking changes.
```

**Comment by timdeschryver** [comment]
Do the schematics need a BEFORE and AFTER? 🤔

**Comment by brandonroberts** [comment]
We don't need before and after for schematics changes.

**Pull request merged**